### PR TITLE
WeekView Sivagnanam maheshwaran

### PR DESCRIPTION
**Pull request by Mahesh1772**

### CODE REVIEW

**Review by NicholasTanYY — COMMENTED** [review]
The coding standards are all adhered to. Could you perhaps put this into src/main/java/ui ?

**Review by clarencepohh — COMMENTED** [review]
LGTM! We could possibly look into integrating time-related functions into a Time package. Let's work out our dependency matters when all important classes are created. 👍

File: src/main/java/seedu/duke/WeekView.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package seedu.duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class WeekView {
```
Comment (kyhjonathan) [inline src/main/java/seedu/duke/WeekView.java:6]: LGTM! We can incorporate this with our UI during our meeting later!

File: src/main/java/seedu/duke/WeekView.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    private LocalDate startOfWeek;
+    private final DateTimeFormatter dateFormatter;
```
Comment (kyhjonathan) [inline src/main/java/seedu/duke/WeekView.java:8]: Good usage of final, preventing any unnecessary accidental changes in the future!

**Pull request merged**